themebuilder/modules/clean-css-options.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
+    // eslint-disable-next-line
```

Comment:
I suggest not to disable eslint for the next line, but add the word `rebase` to ignore list in _eslint-plugin-spellcheck_ config